### ISSUE TITLE
Private keys

### ISSUE THREAD

**Issue by arkadym** [horizontalsystems/unstoppable-wallet-android#7108]
Hi, is there a way to get private key for TRON and TON ? I mean currenly app allow to add those 2 networks to wallet but i will be not able to use same wallet in another place, means i dont have full control over the wallet! 
What i would ask is allow export private keys for TRON and TON network and import existing wallet for those networks , would it be possible ?

**Comment by notpri** [comment]
> Hi, is there a way to get private key for TRON and TON ? I mean currenly app allow to add those 2 networks to wallet but i will be not able to use same wallet in another place, means i dont have full control over the wallet! What i would ask is allow export private keys for TRON and TON network and import existing wallet for those networks , would it be possible ?

you can easily open your wallet in any other wallet app just by your wallet's recovery seed

**Comment by arkadym** [comment]
> > Hi, is there a way to get private key for TRON and TON ? I mean currenly app allow to add those 2 networks to wallet but i will be not able to use same wallet in another place, means i dont have full control over the wallet! What i would ask is allow export private keys for TRON and TON network and import existing wallet for those networks , would it be possible ?
> 
> you can easily open your wallet in any other wallet app just by your wallet's recovery seed

What if my seed has passphrase ?

**Comment by arkadym** [comment]
> > > > Hi, is there a way to get private key for TRON and TON ? I mean currenly app allow to add those 2 networks to wallet but i will be not able to use same wallet in another place, means i dont have full control over the wallet! What i would ask is allow export private keys for TRON and TON network and import existing wallet for those networks , would it be possible ?
> > > 
> > > 
> > > you can easily open your wallet in any other wallet app just by your wallet's recovery seed
> > 
> > 
> > What if my seed has passphrase ?
> 
> try it yourself first

I tried. I created wallet in Unstoppable app, my seed is 24 words and i added passphrase. Obviously when i import in eg Trust Wallet my seed w/o passpharase i see 0 values. If i do import my wallet using private key - i can see proper amounts.

**Comment by arkadym** [comment]
> Trust Wallet doesn't support seed with passphrase. So when you type only the seed, simply you are entering in another account.

Exactly! That is the point i came from, so having private key for TRON & TON wallets will help to solve that!

**Comment by Chvalov** [comment]
[arkadym](https://github.com/arkadym) I was also disappointed by the lack of a feature to generate a private key for Solana, TRON, TON. I developed a python-based tool that allows to generate a private key and use it in other cryptocurrency wallets.
https://github.com/Chvalov/SeedPass2PK

**Comment by Dianaaiym** [comment]
https://github.com/horizontalsystems/unstoppable-wallet-android/issues/9063
